yarn.lock

```diff
@@ -3264,17 +3264,17 @@ rollup-pluginutils@^2.8.1:
   dependencies:
```

Comment:
We don't need both lock files (`package-lock.json` and `yarn.lock`). This file can be deleted. Removing it will help avoid possible conflicts with dependencies and ensure consistency across environments.

Reply:
Yes i'll remove ```yarn.lock``` file